main.js

```diff
@@ -3,76 +3,78 @@ const filterBtn = document.getElementById('filterBtn-js');
 })
 
 const adultCount = document.getElementById('adultCount');
-const childrenCount =document.getElementById('childrenCount');
+const childrenCount = document.getElementById('childrenCount');
 const roomCount = document.getElementById('roomCount');
-const minusBtn = document.querySelectorAll('.minus-filter-btn');
-const plusBtn = document.querySelectorAll('.plus-filter-btn');
+const minusButtons = document.querySelectorAll('.minus-filter-btn');
+const plusButtons = document.querySelectorAll('.plus-filter-btn');
 const childAgeBlock = document.querySelectorAll('.filter-childAgeBlock');
 
 //Adults
 
-minusBtn[0].addEventListener('click', () => {
-  if (adultCount.textContent !== `${0}`) {
+const minAdultValue = '0';
+const maxAdultValue = '30';
+
+minusButtons[0].addEventListener('click', () => {
+  if (adultCount.textContent !== minAdultValue) {
     adultCount.textContent = `${adultCount.textContent - 1}`;
   }
-  if (adultCount.textContent === `${0}`) {
-    minusBtn[0].style.border = '1px solid #CECECE';
-    minusBtn[0].style.color = '#CECECE';
+  if (adultCount.textContent === minAdultValue) {
+    minusButtons[0].style.border = '1px solid #CECECE';
+    minusButtons[0].style.color = '#CECECE';
   }
-  if(adultCount.textContent !== `${30}`) {
-    plusBtn[0].style.border = '1px solid #3077C6';
-    plusBtn[0].style.color = '#3077C6';
+  if(adultCount.textContent !== maxAdultValue) {
+    plusButtons[0].style.border = '1px solid #3077C6';
+    plusButtons[0].style.color = '#3077C6';
   }
   filterBtn.textContent = `${adultCount.textContent} Adults — ${childrenCount.textContent} Children — ${roomCount.textContent} Room`;
 })
 
-plusBtn[0].addEventListener('click', () => {
-  if(adultCount.textContent !== `${30}`) {
+plusButtons[0].addEventListener('click', () => {
+  if(adultCount.textContent !== maxAdultValue) {
     adultCount.textContent = `${Number(adultCount.textContent) + 1}`;
   }
-  if (adultCount.textContent !== `${0}`) {
-    minusBtn[0].style.border = '1px solid #3077C6';
-    minusBtn[0].style.color = '#3077C6';
+  if (adultCount.textContent !== minAdultValue) {
+    minusButtons[0].style.border = '1px solid #3077C6';
+    minusButtons[0].style.color = '#3077C6';
   }
-  if (adultCount.textContent === `${30}`) {
-    plusBtn[0].style.border = '1px solid #CECECE';
-    plusBtn[0].style.color = `#CECECE`;
+  if (adultCount.textContent === maxAdultValue) {
+    plusButtons[0].style.border = '1px solid #CECECE';
+    plusButtons[0].style.color = `#CECECE`;
   }
   filterBtn.textContent = `${adultCount.textContent} Adults — ${childrenCount.textContent} Children — ${roomCount.textContent} Room`;
 })
 
 //Children
-if (childrenCount.textContent === `${0}`) {
-  minusBtn[1].style.border = '1px solid #CECECE';
-  minusBtn[1].style.color = '#CECECE';
-}
 
-minusBtn[1].addEventListener('click', () => {
-  if (childrenCount.textContent !== `${0}`) {
+const minChildrenValue = '0';
+const maxChildrenValue = '10';
```

Comment:
maybe it's better to use numbers here - cause it will be more clear if we talk about js types